framework/decode/dx12_resource_value_mapper.cpp

```diff
@@ -449,9 +449,16 @@ void Dx12ResourceValueMapper::PostProcessExecuteIndirect(DxObjectInfo* command_l
               state_object_extra_info,
               { command_signature_extra_info, argument_buffer_object_info, argument_buffer_offset } });
     }
-    else
+    else if (max_command_count != 0)
```

Comment:
It appears based on the [spec for EI](https://learn.microsoft.com/en-us/windows/win32/api/d3d12/nf-d3d12-id3d12graphicscommandlist-executeindirect#parameters) that if `max_command_count == 0` then EI won't execute any commands, so this conditional could be moved up:
```
if (max_command_count != 0)
{
    if ((count_buffer_object_info != nullptr) && (count_buffer_object_info->object != nullptr))
    {
        // existing code to handle count buffer use...
    }
    else
    {
        // updated code to handle no count buffer...
    }
}